connect/json/src/main/java/org/apache/kafka/connect/json/JsonConverter.java

```diff
@@ -235,18 +235,24 @@ public Object toConnect(final Schema schema, final JsonNode value) {
     private final JsonDeserializer deserializer;
 
     public JsonConverter() {
+        this(true);
+    }
+
+    public JsonConverter(boolean enableModules) {
```

Comment:
please add a javadoc comment specifying that this ctor is visible only for benchmarking. we add similar comments when we expose a new ctor/method for testing.